pyccel/ast/internals.py

```diff
@@ -173,6 +179,14 @@ def step(self):
         """
         return self._step
 
+    @property
+    def slice_type(self):
+        """ The type of the slice (Range or Index)
+        Range <=> [..., :, ...]
+        Index <=> [..., 3, ...]
+        """
+        return self._slice_type
```

Comment:
On line 144 you call it `Element`, not `Index`

Reply:
https://github.com/pyccel/pyccel/pull/1299/commits/7a51001eff28302d544896356e850fa6f21bf8e9